Sources/NetworkProtection/WireGuardKit/PacketTunnelSettingsGenerator.swift

```diff
@@ -113,25 +113,26 @@ final class PacketTunnelSettingsGenerator {
             networkSettings.mtu = NSNumber(value: mtu)
         }
 
-        let (ipv4Addresses, ipv6Addresses) = addresses()
-        let (ipv4IncludedRoutes, ipv6IncludedRoutes) = includedRoutes()
+        let addresses = Self.routes(from: tunnelConfiguration.interface.addresses)
+        let excludedRoutes = Self.routes(from: tunnelConfiguration.interface.exclusions)
 
-        let ipv4Settings = NEIPv4Settings(addresses: ipv4Addresses.map { $0.destinationAddress }, subnetMasks: ipv4Addresses.map { $0.destinationSubnetMask })
-        ipv4Settings.includedRoutes = ipv4IncludedRoutes
-        ipv4Settings.excludedRoutes = Self.ipv4ExcludedRoutes
+        let ipv4Settings = NEIPv4Settings(addresses: addresses.ipv4.map { $0.destinationAddress }, subnetMasks: addresses.ipv4.map { $0.destinationSubnetMask })
+        ipv4Settings.includedRoutes = [.default()] // 0.0.0.0/0
+        ipv4Settings.excludedRoutes = excludedRoutes.ipv4
```

Comment:
Is this the configuration just for when the kill-switch is enabled? or it's now the configuration always

Reply:
Always, dns server added to Exclusion list (as it was before) if the kill switch is off

Reply:
Is not that a change in behavior?
When the kill-switch is OFF we should comply with [this](https://app.asana.com/0/0/1204105983557593/f)

Reply:
It seems smart enough to not route the local network requests through the tunnel, in fact the excludeLocalNetworks flag didn't work for me, but I'm yet to validate this next week

Reply:
The exclusions are now provided from the main app, something to handle in the follow up project

Reply:
I agree the routing table shouldn't be affected in this PR as we agreed to the kill switch feature not affecting existing features.

Let's re-enable the routing table we had when the kill switch is off.

Reply:
since we have an exclusion list planned and hardcoded items added in Debug menu, this is now set from the NetworkConnectionController